pandas/core/frame.py

```diff
@@ -3373,16 +3381,21 @@ def _set_item(self, key, value):
 
         self._set_item_mgr(key, value)
 
-    def _set_value(self, index, col, value, takeable: bool = False):
+    def _set_value(
+        self, index: IndexLabel, col, value: Scalar, takeable: bool = False
+    ) -> None:
         """
         Put single value at passed column and index.
 
         Parameters
         ----------
-        index : row label
-        col : column label
+        index : Label
```

Comment:
even though this function is internal, the docstring should not use typing alias, but numpydoc conventions